pallets/unified-accounts/src/migration.rs

```diff
@@ -16,14 +16,9 @@
 // You should have received a copy of the GNU General Public License
 // along with Astar. If not, see <http://www.gnu.org/licenses/>.
 
-use super::{Config, Pallet, Weight};
+use super::{Config, Pallet};
 use astar_primitives::evm::EvmAddress;
-use frame_support::{
-    pallet_prelude::OptionQuery,
-    storage_alias,
-    traits::{Get, OnRuntimeUpgrade},
-    Blake2_128Concat,
-};
+use frame_support::{pallet_prelude::OptionQuery, storage_alias, Blake2_128Concat};
 
 #[storage_alias]
 type EvmToNative<T: Config> = StorageMap<
```

Comment:
don't think those are needed anymore

Reply:
Good catch, I'll remove the entire file.